test/features/data_my_personal_data_usage.feature

```diff
@@ -1,19 +1,17 @@
-Feature: API endopoint allowing users to check who had read their personal data in the Digital Registries database.
+Feature: API endpoint allowing users to check who had read their personal data in the Digital Registries database.
   Request endpoint: GET /data/MyPersonalDataUsage/1.0/
 
   Scenario: The user receives a list with all records that had read his personal data in the Digital Registries database
-    Given The user with id=282392302 wants to check who had read his personal data in the Digital Registries database
-    And In the database, there is a list of users who read the user with id=282392302 personal data
-    When The user with id=282392302 triggers an action to receive a list of users who read his personal data
+    Given The user wants to check who had read his personal data in the Digital Registries database and there is a list of users who did it
+    When The user triggers an action to receive a list of users who read his personal data
     Then The user receives a list with all records
 
   Scenario: The user receives an empty list of records that had read his personal data from the Digital Registries database
-    Given The user with id=748382347 wants to check who had read his personal data in the Digital Registries database
-    And In the database there is an empty list of users who read the user with id=748382347 personal data
-    When The user with id=748382347 triggers an action to receive a list od users who read his personal data
-    Then The user with id=748382347 receives an empty list because there are no records in the database about other users who read his personal data
+    Given The user wants to check who had read his personal data in the Digital Registries database and there is an empty list of users who did it
+    When The user triggers an action to receive a list of users who read his personal data
+    Then The user receives an empty list because there are no records in the database about other users who read his personal data
 
   Scenario: The user is not able to receive data from the Digital Registries database because of an invalid request
-    Given The user with id=3782347 wants to check who had read his personal data in the Digital Registries database
-    When The user with id=3782347 triggers an action to receive a list od users who read his personal data
+    Given The user wants to check who had read his personal data in the Digital Registries database
+    When The user triggers an action to receive a list of users who read his personal data with an invalid request
```

Comment:
maybe also 
```The user triggers an action with an invalid request to receive a list of users who have read his personal data```

Reply:
The meaning of the sentence is the same. I'm not sure if this change is necessary

Reply:
The suggested is version is less confusing, but what does 'triggering an action with an invalid request' even mean?

I imagine the user does not 'trigger' anything because the request is invalid and he gets a validation error or something along those lines. 

I would use this wording:

`The user sends an invalid request in an attempt to retrieve the list of users who have read his personal data`

Let me know if that's too much for Gherkin and do even need the 'triggers an action' part.

Btw I would use retrieve instead of receive as the word describing the action, since they want to get it immediately, not receive it later.